client/keys/add_test.go

```diff
@@ -127,6 +127,77 @@ func Test_runAddCmdBasic(t *testing.T) {
 	require.Error(t, cmd.ExecuteContext(ctx))
 }
 
+func Test_runAddCmdMultisig(t *testing.T) {
```

Comment:
Let's make the name of this test reflect quickly what it does
Test_runAddCmdMultisigDupKeys or even more succinctly Test_cmdCatchDuplicateKeysInMultisig

Reply:
Done